.github/workflows/release.yml

```diff
@@ -31,12 +31,13 @@ jobs:
           pip install yq
 
           cd /usr/bin
-          curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/kustomize/v4.0.4/hack/install_kustomize.sh"  | bash
+          curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/kustomize/v4.0.5/hack/install_kustomize.sh" | bash
```

Comment:
well, I would let makefile install the kustomize. Otherwise, released deployment can be different from what we have in main branch after we upgrade kustomize in one place

Reply:
True, but currently make-release.sh doesn't depend on make at all and calls the script directly :/ 

I wonder if make is even installed in the ubuntu runner

Reply:
Fixed by moving download of kustomize into generate-deployment.sh